cmd/amazon-cloudwatch-agent/amazon-cloudwatch-agent.go

```diff
@@ -139,6 +140,36 @@ func reloadLoop(
 			}
 		}(ctx)
 
+		if envConfigPath, err := getEnvConfigPath(*fConfig, *fEnvConfig); err == nil {
+			// Reloads environment variables when file is changed
+			go func(ctx context.Context, envConfigPath string) {
+				var previousModTime time.Time
+				ticker := time.NewTicker(30 * time.Second)
+				defer ticker.Stop()
+				for {
+					select {
+					case <-ticker.C:
+						if info, err := os.Stat(envConfigPath); err == nil && info.ModTime().After(previousModTime) {
+							if err := loadEnvironmentVariables(envConfigPath); err != nil {
+								log.Printf("E! Unable to load env variables: %v", err)
```

Comment:
Are we only going to log when we are unable to load environment variables? I guess it would make sense that if we encountered an error here, we wouldn't modify the log level.